### PR TITLE
Bug 1804441: Provide external log link for pod

### PR DESCRIPTION
**Pull request by lukas-vlcek**
Setup external log links for both admin and non-admin users for pods.

### CODE REVIEW

**Comment by igor-karpukhin** [comment]
@lukas-vlcek please update the BZ to get this PR merged.

File: pkg/k8shandler/visualization.go [hunk 1]
```diff
@@ -432,31 +435,54 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaConsoleExternal
...
+	links := []*consolev1.ConsoleExternalLogLink{consoleExternalLogLinkForNodesAndContainers, consoleExternalLogLinkForApps}
 
-	err = clusterRequest.Create(consoleExternalLogLink)
-	if err != nil && !errors.IsAlreadyExists(err) {
-		return fmt.Errorf("Failure creating Kibana console external log link for %q: %v", cluster.Name, err)
+	for _, externalLink := range links {
+		utils.AddOwnerRefToObject(externalLink, utils.AsOwner(cluster))
+		// In case the object already exists we delete it first
+		if err = clusterRequest.RemoveConsoleExternalLogLink(externalLink.ObjectMeta.Name); err != nil {
+			return
```
Comment (huikang) [inline pkg/k8shandler/visualization.go:479]: Missing `fmt.Errorf(...)`?

**Review by huikang — COMMENTED** [review]
Hi, I am learning the code base by studying your PR. Thanks.

**Comment by jcantrill** [comment]
Closing per 4.3 BZ was closed

**Pull request closed without merging**